tests.js

```diff
@@ -3886,7 +3888,7 @@ window.Specs = {
 	"pointerevents": {
 		"title": "Pointer Events",
 		"links": {
-			"tr": "pointerevents2",
+			"tr": "pointerevents",
 			"dev": "pointerevents",
```

Comment:
Pointer Events Level 2

TR : https://w3c.github.io/pointerevents/
DEV : https://www.w3.org/TR/pointerevents2/

Reply:
I just saw that the draft is already at level 3 and that https://www.w3.org/TR/pointerevents3/ is a thing.
Should i link to `pointerevents3` instead?

Reply:
Level 1 exist too: https://www.w3.org/TR/pointerevents1/
We take the functionalities in the oldest version.